package.json

```diff
@@ -36,7 +36,8 @@
     "test:coverage": "jest --coverage",
     "eslint": "eslint --ext .tsx,.js ./src/",
```

Comment:
@cdcabrera good catch, fixed that. I also had to add some eslint-ignore comments in src/common/helpers.ts because there's a ton of `any` usage in there and I think maybe fixing that is outside the scope of this PR